OrchardCMS/OrchardCore
# Fix case insensitive dictionary problem (Lombiq Technologies: OCORE-197)

Fixes #16585

Easiest way to test:
1. Create a new site with Blog recipe
2. Go to Admin > Design > Templates
3. Click "Add Template"
4. Name it "Menu", "MENU", or "mEnU", set some custom content.
5. Load the frontend and observe the overridden menu.
